mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp

```diff
@@ -416,6 +425,12 @@ int CMomentumStickybombLauncher::DetonateRemoteStickybombs()
         }
     }
 
+    // If two stickies have been det, mark time for buffer
+    if (stickyBombsAffecting[1])
+    {
+        m_fl2DetCooldownBeginTime = gpGlobals->curtime;
+    }
+    
```

Comment:
Should `m_fl2DetCooldownBeginTime` be reset here if not detonating two?

Reply:
It might be more readable that way, but it doesn't need to functionally. It works as a time stamp to check if gpGlobals->curtime - m_fl2DetCooldownBeginTime  is less than the buffer duration. I can go ahead and add that change though, the logic still works fine with that.